docs/tutorials/running-the-dal-as-a-weeklynet-baker-in-5-steps.md

```diff
@@ -0,0 +1,20 @@
+# Running the DAL as a Weeklynet baker, in 5 steps
+
+Tezos' Data-Availability Layer (DAL for short), which we presented [in a previous article](https://research-development.nomadic-labs.com/data-availability-layer-tezos.html), is a key component of our scalability roadmap for Tezos. In a nutshell, the DAL is about increasing the bandwidth available for Tezos smart rollups thanks to a new parallel P2P network on which rollup could connect to fetch inputs but without compromising their security and in particular the possibility for any participant to detect fraud attempts from rollup operators and refute them in the Tezos protocol.
```

Comment:
`on which rollup could connect` -> `on which rollups could connect`

Reply:
and I find the end of the paragraph a little bit technical already.

Reply:
Typo fixed in d5f17df.

For the technicality of the end of the paragraph, what about moving it to a footnote?

Reply:
Removed the technical part in 95c3fae.